README.md

```diff
@@ -129,6 +129,9 @@ license_scanning:
     TRIVY_YAML: './frontend/custom-trivy.yaml'
 ```
 
+Keep in mind that if you detect a package with badly configured licences like `pipenv`, maybe you can 
```

Comment:
Den würde ich umformulieren:
```suggestion
Some packages in the python world report the full license text as their license. Until that is fixed upstream these packages will be reported having unknown licenses and show up as an error.  
As of October 2023, we know at least `pipenv` and `traitlets` do this.  
Because pipenv is only needed during setup, you can easily remove that requirement from your pipfile without any disadvantage.
```